onnxscript/ir/_core.py

```diff
@@ -508,7 +511,19 @@ def _quoted(string: str) -> str:
 
 
 class Node(_protocols.NodeProtocol, _display.PrettyPrintable):
-    """IR Node."""
+    """IR Node.
+
+    When the node is initialized, it does not belong to any graph. It is the
+    responsibility of the caller to add the node to a graph, by calling :func:`Graph.absorb_nodes`.
```

Comment:
`absorb_nodes` this seems new to me, maybe I missed it in previous PRs.

Reply:
Updated.